src/models/Room.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, model, Types } from 'mongoose';
+
+interface Room {
+    name: string;
+    sections: Types.Array<string>;
```

Comment:
What is the difference between a mongoose Array and a normal js Array?

What specifically is sections supposed to represent here?